core/frontend/src/components/health/GpsTrayMenu.vue

```diff
@@ -40,6 +98,85 @@ export default Vue.extend({
     }
   },
   computed: {
+    values():
+      Array<
+        {
+          name: string,
+          value: string,
+          tooltip?: string,
+          icon: string,
+          link?: string,
+          link_icon?: string
+        }> {
+      if (!this.gps_raw_int) {
+        return []
+      }
+      const values = [
+        {
+          name: 'Position',
+          value: `${this.lattitude.toFixed(2)}º / ${this.longitude.toFixed(2)}º`,
+          tooltip: 'Position coordinates',
+          icon: 'mdi-map-marker',
+          link: this.map_link,
+          link_icon: 'mdi-map-search',
+        },
+        {
+          name: 'Altitude',
+          value: `${(this.gps_raw_int.alt / 1000).toFixed(2)} m`,
+          tooltip: 'Altitude in meters',
+          icon: 'mdi-arrow-collapse-up',
+        },
+        {
+          name: 'Satellites',
+          value: `${this.satellites_number}`,
+          tooltip: 'Number of satellites',
+          icon: 'mdi-satellite-variant',
+        },
+        {
+          name: 'Fix',
+          value: `${this.connection_description}`,
+          tooltip: 'GPS connection status',
+          icon: 'mdi-crosshairs-gps',
+        },
+        {
+          name: 'HDOP',
+          value: `${(this.gps_raw_int.eph / 100).toFixed(2)}`,
+          tooltip: 'Horizontal Dilution of Precision',
```

Comment:
```suggestion
          tooltip: 'Horizontal position uncertainty of GPS signal',
```
I feel like "HDOP" and "Horizontal Dilution of Precision" are similarly easy to _remember_ a meaning of, but hard to interpret for the first time - it seems more useful for the tooltip to describe the meaning more directly.

By specifying "GPS signal" we can hopefully make it clearer that it's not about the vehicle's position estimate, but rather only related to the signal it's receiving from the GPS subsystem.